### ISSUE TITLE
The occ:permissions command can't display lines such as PATH.

### ISSUE THREAD

**Issue by Bables55** [nextcloud/groupfolders#550]
I ran permissions with the occ command, but I can't see the results, such as PATH.

```
occ groupfolders:permissions 1
+------+------------+-------------+
| Path | User/Group | Permissions |
+------+------------+-------------+
```

Only the above header is displayed.
“Advanced Permissions” is enabled for each account.
It is the same even if many GroupFolders folders are created.
"occ groupfolders:list" is displayed normally.

What should I do to make this display correctly?

--- 
CentOS 7.6, PHP7.3.7(tried 7.1), MariaDB 10.2.26, nginx 1.14.2, Groupfolders 4.0.5

**Comment by Bables55** [comment]
Is there a way to fully initialize Groupfolders?

**Comment by jstone1984** [comment]
@Bables55 I have encountered the same problem. Have you solved it?

**Comment by Bables55** [comment]
I have not solved yet. So I want to start over, but I don't know how to handle the wreckage database when I delete the app.

**Comment by jstone1984** [comment]
Thank you for your reply. I think about it again. If you solve the problem, please let me know, thank you!

**Comment by Bables55** [comment]
When I update to 4.1.2, there is no change in the following status.
Is this condition abnormal or normal?

```
occ groupfolders:permissions 1
+------+------------+-------------+
| Path | User/Group | Permissions |
+------+------------+-------------+
```

**Comment by Bables55** [comment]
This issue cannot be solved with NC17 and Group folder 5.0.5.

**Comment by fschrempf** [comment]
@Bables55 What do you expect to see instead of the empty table. Did you set up any ACL rules? If you didn't create any rules, there won't be anything in the table :wink:
